.gitlab-ci.yml

```diff
@@ -4,6 +4,8 @@ variables:
   CURRENT_CI_IMAGE: 25
   BUILD_STABLE_REGISTRY: '486234852809.dkr.ecr.us-east-1.amazonaws.com'
   CI_IMAGE: '$BUILD_STABLE_REGISTRY/ci/$APP:$CURRENT_CI_IMAGE'
+  GIT_REPOSITORY: 'git@github.com:DataDog/browser-sdk.git'
```

Comment:
Could `CI_REPOSITORY_URL` predefined env variable work? or is it the gitlab url?

Reply:
Sadly it references the gitlab url

Reply:
kinda make sense